R/utils.R

```diff
@@ -1,3 +1,48 @@
+register_functions_to_cluster <- function(cluster) {
+    snow::clusterExport(cluster, list(
+        'proc.cdf',
+        'prof.to.features',
+        'load.lcms',
+        'adaptive.bin',
+        'find.turn.point',
+        'msExtrema',
+        'find_local_maxima',
+        'combine.seq.3',
+        'cont.index',
+        'interpol.area',
+        'load_file',
+        'load_data',
+        'plot_raw_profile_histogram',
+        'compute_mass_values',
+        'compute_densities',
+        'compute_breaks',
+        'compute_breaks_3',
+        'compute_boundaries',
+        'increment_counter',
+        'rm.ridge',
+        'compute_delta_rt',
+        'bigauss.mix',
+        'bigauss.esti',
+        'rev_cum_sum',
+        'compute_bounds',
+        'validate_inputs',
+        'preprocess_bandwidth',
+        'preprocess_profile',
+        'compute_gaussian_peak_shape',
+        'compute_chromatographic_profile',
+        'compute_dx',
+        'compute_initiation_params',
+        'compute_e_step',
+        'compute_start_bound',
+        'compute_end_bound',
+        'compute_bounds',
+        'compute_scale',
+        'span'
+    ))
+    snow::clusterEvalQ(cluster, library("dplyr"))
+}
```

Comment:
Should have introduced this long time ago instead of exporting only the used functions in every place - this is so much more convenient!